apps/eth-wrapper/README.md

```diff
@@ -0,0 +1,7 @@
+# Eth-Wrapper
+
+## Usage
+
+Simple safe app to wrap ETH or unwrap WETH.
+
+Add the Safe app at your own risk. This is hosted: [https://jpalvarezl.github.io/eth-wrapper/](https://jpalvarezl.github.io/eth-wrapper/)
```

Comment:
I would make clear that this is not necessarily the same version as the one in this repo (unless you keep your repo in sync with this one)

Reply:
I will remove this, as I suppose having this deployed in my personal github pages is not ideal